_data/books.yml

```
@@ -28,3 +28,26 @@
     - title: |
         The Culture Map - Breaking Through the Invisible Boundaries of Global Business by Erin Meyer
       comments: Another amazing book around team culture.
+
+- index: 12 # mentor id 12
+  books:
+  I never stop reading, trying to learn more, to be better at what I do. Here are some of my favourites, in no particular order.
```

This session does not exist, if you want to add you have to include in the html page. For example: 

```suggestion
  description: | 
  I never stop reading, trying to learn more, to be better at what I do. Here are some of my favourites, in no particular order.
```

---

Thank you for your comment @dricazenck. I think we shouldn't add this sentence to avoid changing the system there. Only book titles and descriptions should be included. So, we can remove this sentence.